packages/next-typescript/index.js

```diff
@@ -35,23 +35,15 @@ module.exports = (nextConfig = {}) => {
         })
       }
 
+      if (!defaultLoaders.babel.options.babelrc) {
+        defaultLoaders.babel.options.presets.unshift("@babel/preset-typescript")
```

Comment:
Lets have the user explicitly configure this in .babelrc, regardless of if they have one or not.